apps/pie-storybook/stories/pie-notification.stories.ts

```diff
@@ -16,11 +15,23 @@ type NotificationStoryMeta = StoryMeta<NotificationProps>;
 const defaultArgs: NotificationProps = {
     isOpen: true,
     variant: 'neutral',
+    isDismissible: true,
     isCompact: false,
-    slot: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet tincidunt est, vitae vulputate turpis. Cras pretium venenatis elementum. Duis tristique neque non varius tempor. In hac habitasse platea dictumst. Aenean accumsan vehicula urna. Cras fringilla sed ipsum nec dignissim. Aliquam sit amet ullamcorper ligula.',
+    slot: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus sit amet tincidunt est, vitae vulputate turpis. Cras pretium venenatis elementum. Duis tristique neque non varius tempor. In hac habitasse platea dictumst. Aenean accumsan vehicula urna.',
     heading: 'Title',
     headingLevel: 'h2',
     hideIcon: false,
+    leadingAction: {
+        text: 'Confirm',
+        ariaLabel: 'Descriptive confirmation text',
+        onClick: () => { console.log('Leading action triggered'); },
```

Comment:
We could leverage storybook actions here instead of console logs - entirely up to you but there should be examples in the `pie-input` stories if you're interested